clab/clab.go

```diff
@@ -626,6 +580,21 @@ func (c *CLab) ListNodesContainers(ctx context.Context) ([]runtime.GenericContai
 	return containers, nil
 }
 
+// ListNodesContainersIgnoreNotFound lists all containers based on the nodes stored in clab instance, ignoring errors for non found containers
+func (c *CLab) ListNodesContainersIgnoreNotFound(ctx context.Context) ([]runtime.GenericContainer, error) {
+	var containers []runtime.GenericContainer
+
+	for _, n := range c.Nodes {
+		cts, err := n.GetContainers(ctx)
+		if err != nil {
+			continue
+		}
+		containers = append(containers, cts...)
+	}
+
+	return containers, nil
+}
+
```

Comment:
@steiler  can the error be checked here to be exactly "not found" and not just any error?

Reply:
yes, but I think we need to then implement a "not found" error and make the runtimes convert their respective not found to  a clab generic "not found" and handle that.
Can you maybe take that action point?